tests/integrationtest/r/ddl/partition.result

```diff
@@ -177,3 +177,180 @@ t	CREATE TABLE `t` (
   KEY `b` (`b`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
 PARTITION BY KEY (`a`) PARTITIONS 5
+set character_set_connection=gbk;
+drop table if exists t;
+create table t (col1 varbinary(16) unique key) partition by list columns(col1)
+(partition p0 values in ('你好', '我好'), partition p1 values in ('大家好'), partition p2 DEFAULT);
+show create table t;
+Table	Create Table
+t	CREATE TABLE `t` (
+  `col1` varbinary(16) DEFAULT NULL,
+  UNIQUE KEY `col1` (`col1`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
+PARTITION BY LIST COLUMNS(`col1`)
+(PARTITION `p0` VALUES IN (_binary 0xc4e3bac3,_binary 0xced2bac3),
+ PARTITION `p1` VALUES IN (_binary 0xb4f3bcd2bac3),
+ PARTITION `p2` DEFAULT)
+insert into t values ("你好");
+select * from t partition(p0);
+col1
+���
```

Comment:
It seems @Defined2014 have modify mysql-test to display the binary human-readable somehow

Reply:
@YangKeao Please change it to `select hex(col1) from t partition(p0);`.